### DISCUSSION TITLE
How to use for plain Git installation

### DISCUSSION THREAD

**Discussion by divakarsingh** [renovatebot/renovate#25154] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi, I just started studying Renovate.
I have some basic questions.

I have my own Git repo, not using any third party git service like github, gitlab, azure devops etc.
How can I use Renovate?

Also would like to understand, does renovate scan java based libraries for update using pom files, or can it scan other dependencies also like databases etc, from the used libraries in the app source code?

Thank in advance!

### What you tried so far.

NA

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You cannot
